observability-kit-starter/src/main/java/com/vaadin/observability/ObservabilityHandler.java

```diff
@@ -65,6 +90,78 @@ static ObservabilityHandler ensureInstalled(UI ui) {
         return newObservabilityHandler;
     }
 
+    private final InstrumentationConfig config;
+    private SpanExporter exporter = null;
+
+    public ObservabilityHandler() {
+        this.config = InstrumentationConfig.get();
+        String exporterType = config.getString("otel.traces.exporter", "otlp");
```

Comment:
Is the aim of this block to create a `SpanExporter` equal to the one created by the agent during bootstrap?